packages/contracts-bedrock/test/DeployImplementations.t.sol

```diff
@@ -103,7 +110,8 @@ contract DeployImplementationsOutput_Test is Test {
             OptimismMintableERC20Factory(makeAddr("optimismMintableERC20FactoryImpl"));
         DisputeGameFactory disputeGameFactoryImpl = DisputeGameFactory(makeAddr("disputeGameFactoryImpl"));
 
-        vm.etch(address(opsm), hex"01");
+        vm.etch(address(opsmProxy), address(opsmProxy).code);
+        vm.etch(address(opsmImpl), hex"01");
```

Comment:
I think `opsmImpl` is an unused var that we can delete?

Reply:
I'm using it above to set the proxy's impl: https://github.com/ethereum-optimism/optimism/pull/11875/files#diff-6f6c4154ee2681e7330cc0b8bba542efcd65e748af84aea7ffa5b69ebc44df92R97

Reply:
Yea, but we don't need to set that in this test do we? It's just an empty address with no code so seems equivalent to the zero address

Reply:
Ah, the line:
```solidity
assertEq(address(opsm), address(dio.opsm()), "50");
```
Checks if the opsm has an implementation contract with code at it.

Reply:
I see, thanks!